cassandra/1.cql

```diff
@@ -1,13 +1,13 @@
 CREATE KEYSPACE devgym
   WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
-
+/
```

Comment:
Shows `Invalid syntax at line 1, char 1` if I try this script manually, but seems to work anyway. Working solution :+1: